<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSetupWin</class>
 <widget class="QMainWindow" name="AdvancedSetupWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced_Setup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>331</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>bodyBuild</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>controlers shape fix</string>
      </property>
      <widget class="QPushButton" name="rotate_Btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>rotate</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rotate_rollBack_Btn">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>roll back</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>mirror ik control </string>
      </property>
      <widget class="QPushButton" name="mirror_Btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>mirror</string>
       </property>
      </widget>
      <widget class="QPushButton" name="mirror_rollBack_Btn">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>roll back</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>add main control</string>
      </property>
      <widget class="QPushButton" name="addWorldCtl_Btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>add world control</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rollback_Btn">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>roll back</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>edit Arm follow options</string>
      </property>
      <widget class="QPushButton" name="edit_follow_Btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>edit follow</string>
       </property>
      </widget>
      <widget class="QPushButton" name="edit_follow_rollback_Btn">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>roll back</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>IK_subCTL</string>
      </property>
      <widget class="QPushButton" name="subCTL_Btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>subCTL</string>
       </property>
      </widget>
      <widget class="QPushButton" name="subCTL_rollBack_Btn">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>roll back</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>dual Quaternion nonRigid Setup</string>
      </property>
      <widget class="QPushButton" name="nonRigid_On_Btn">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
      <widget class="QPushButton" name="nonRigid_Off_Btn">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>OFF</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="allAtOnce_Btn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>430</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>All at Once</string>
      </property>
     </widget>
     <widget class="QPushButton" name="allAtOnce_rollBack_Btn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>430</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>roll back</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="IKRoot_transCTL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>IKRoot_transCTL</string>
      </property>
      <widget class="QPushButton" name="transCTL_Btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>transCTL</string>
       </property>
      </widget>
      <widget class="QPushButton" name="transCTL_rollBack_Btn">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>roll back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>fixBuild</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>legFlatten</string>
      </property>
      <widget class="QPushButton" name="legFlatten_Btn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>legFaltten</string>
       </property>
      </widget>
      <widget class="QPushButton" name="legFlattenRollBack_Btn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>legFlattenRollBack</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="legGoToBuildPose_Btn">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>leg - GoToBuildPose</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>faceBuild</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>301</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>headRIG</string>
      </property>
      <widget class="QPushButton" name="BS_Connect_Btn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>251</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>BS_Connect</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>251</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>select &quot;oneMesh_&quot;(with_ASBS) And Press</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>281</width>
         <height>80</height>
        </rect>
       </property>
       <property name="title">
        <string>headRIG_FIX</string>
       </property>
       <widget class="QPushButton" name="headRIG_FIX_head_Btn">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>headRIG</string>
        </property>
       </widget>
       <widget class="QPushButton" name="headRIG_FIX_body_Btn">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>bodyRIG</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>301</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>attacher_rig</string>
      </property>
      <widget class="QPushButton" name="attacher_rig_Head_Btn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>headRIG</string>
       </property>
      </widget>
      <widget class="QPushButton" name="attacher_rig_Body_Btn">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>bodyRIG</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="attacher_rig_lineEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>upperCheek</string>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>329</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
